config/initializers/ab_tests.rb

```diff
@@ -1,5 +1,6 @@
 require 'ab_test_bucket'
 
+# When adding an A/B test, be sure to add related analytics to AbTestAnalyticsConcern
```

Comment:
Good note. I wonder if there is more we could do to encapsulate an A/B test definition in this file.

Reply:
Yes, there is definitely more to do for documentation, but this was the immediate need/fix.